### DISCUSSION TITLE
Is Provider Collateral lost if the deal is rejected

### DISCUSSION THREAD

**Discussion by lbj2004032** [filecoin-project/boost#1414] [General]
![image](https://user-images.githubusercontent.com/101935091/235941406-2c3a2375-72f2-4211-8a0b-db7e7007721d.png)
![image](https://user-images.githubusercontent.com/101935091/235948782-65188884-645c-4daf-acde-6835e4ed3550.png)

Do these error deals mean a loss of 0.01 Fil

**Comment by LexLuthr** [comment]
@lbj2004032 I would recommend using [filecoin slack](https://filecoinproject.slack.com/archives/C03CKDLEWG1) for question instead of opening Github discussion. You would get help from the whole community there.
I don't see any error or bug in this discussion. You won't lose the 0.01 Fil. That is just the detail about the proposal. Since you never send Publish Message, no amount is deducted. 
I am closing this discussion.

**Reply by lbj2004032** [reply]
it unable join filecoin slack
![image](https://user-images.githubusercontent.com/101935091/236069583-4ef39871-e33d-4a85-bba7-d9c3ae6a1e27.png)

**Reply by LexLuthr** [reply]
You can use https://filecoin.io/slack to sign up
